src/styles/blocks/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+  
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+  
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+  
+.stars.stars--0 .stars__star,
+.stars.stars--1 :not(:first-child),
+.stars.stars--2 :nth-last-child(-n+3),
+.stars.stars--3 :nth-last-child(-n+2),
+.stars.stars--4 :nth-last-child(-n+1),
+.stars.stars--5 .stars__star {
+  background-image: url("/images/star.svg");
```

Comment:
let's avoid too many nth-child, and set background-image as default for stars__star class